packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -130,21 +136,53 @@ function Thread( {
 	const restReplies = !! replies.length ? replies.slice( 0, -1 ) : [];
 
 	return (
+		// Disable reason: role="listitem" does in fact support aria-expanded.
+		// eslint-disable-next-line jsx-a11y/role-supports-aria-props
```

Comment:
ESLint warns `aria-expaned` can't be used with `role="listitem"`, but I think this warning is incorrect.